deps/v8/src/runtime/runtime-i18n.cc

```diff
@@ -627,7 +627,7 @@ RUNTIME_FUNCTION(Runtime_CreateBreakIterator) {
 
   local_object->SetInternalField(0, reinterpret_cast<Smi*>(break_iterator));
   // Make sure that the pointer to adopted text is NULL.
```

Comment:
`NULL` is just `0` with a `nullptr_t` type, which clang complains about. I didn't change the comment, because semantically, I thought that was still more meaningful; I've always thought of `NULL` as <code>void&nbsp;*</code>, but these days it's more fancy.